src/shared/canvas/MyData/types.ts

```diff
@@ -0,0 +1,48 @@
+import { MenuProps } from 'antd';
```

Comment:
it's not new, it was there from day 1
just I refactor the types since there was many and used in multiple places, 
MyData, MyDataHeader, MyDataTable